aspnetcore/mvc/views/tag-helpers/builtin/resources/AnchorTagHelper.md

```diff
@@ -8,41 +8,33 @@ By [Peter Kellner](http://peterkellner.net)
 
 The ```AnchorTagHelper``` enhances the html anchor (`<a ... ></a>`) tag. A new set of attributes are defined that work with the anchor tag. The link generated (on the href tag) is based on a combination of these new attributes that work together to form the final redirect URL. That URL can include an optional protocol such as https.
 
-For reference, the following ASP.NET ```startup.cs``` and ```SpeakerController.cs``` are defined as you would expect in a default Visual Studio .Net Core Web Project.
+For reference, the following ASP.NET ```SpeakerController.cs``` is defined as you would expect in a default Visual Studio .Net Core Web Project.
```

Comment:
* ".Net" to ".NET"
* TH's work outside of VS, so I'd drop mentioning the IDE
* The `SpeakerController.cs` file isn't a default project file, so I wouldn't expect to see it :grin: ... but I know what you mean. I think you just need to rephrase it a little. You could just go with the old "The following example shows ... {insert description here of what it shows}:"